server/api/controller/plan.js

```diff
@@ -64,9 +63,6 @@ class PlanController extends Controller {
 			type: 'Polygon',
 			coordinates: [points]
 		};
-		if (!GJV.valid(geojson)) {
```

Comment:
I think I added that and when I tested it with invalid geojsons it didnt catch it
do you find it useful?

Reply:
i think you're confusing this with another test, as this one seems to have been [added in february](https://github.com/meirim-org/meirim/commit/b6f4f525d07b5860ca1df41e8185d972552a1b26#diff-3e9d9092c88f5d1e691049a150a6ef70e24feecb4a7d602bfaef31190d6925efR67).
i'm not sure what exact validations it runs or what was the problem it was introduced because of, but let's keep it.